apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx

```diff
@@ -53,10 +60,9 @@ function BackToInternalDomain({
 
   useEffect(() => {
     (async () => {
-      const lastScreen = (await AsyncStorage.getItem("last-screen")) || "";
-      setButtonText(lastScreen === "compare_providers" ? t("common.quote") : manifest.name);
+      setButtonText(btnText);
     })();
-  }, [manifest.id, manifest.name, t]);
+  }, [btnText, manifest.id, manifest.name, t]);
```

Comment:
This array has extra dependencies for no reason, only `[btnText]` will be enough